tests/utils.py

```diff
@@ -197,7 +197,11 @@ def generate_user_instance(
 
 def validate_dict_response(response, expected_code, expected_fields):
     assert response.status_code == expected_code
-    assert response.content_type == 'application/json'
+    # The beauty of standards is that there as so many to choose from
+    assert (
+        response.content_type == 'application/json'
```

Comment:
slightly uncomfortable about this. Is this a difference we want to check in some tests?